backend/onedrive/onedrive.go

```diff
@@ -142,6 +149,20 @@ func init() {
 			case "siteid":
 				fmt.Printf("Paste your Site ID here> ")
 				siteID = config.ReadLine()
+			case "url":
+				fmt.Println("Example: \"https://contoso.sharepoint.com/sites/mysite\" or \"mysite\"")
+				fmt.Printf("Paste your Site URL here> ")
+				siteURL := config.ReadLine()
+				re := regexp.MustCompile(`https://.*\.sharepoint.com/sites/(.*)`)
+				match := re.FindStringSubmatch(siteURL)
+				if len(match) == 2 {
+					relativePath = "/sites/" + match[1]
+				} else {
+					relativePath = "/sites/" + siteURL
```

Comment:
This looks strange adding `https://whatever.com` onto `/sites/` - is that correct?

Reply:
In short:
+ If user enter something like `https://contoso.sharepoint.com/sites/mysite`, then we find out the `mysite`
+ If not, then assuming the user already find out the `mysite` part, just use it as it is

Since something user could easily remember the `mysite` from `https://contoso.sharepoint.com/sites/mysite`, no need for entering the whole URL.